civictechprojects/static/css/partials/_SplashScreen.scss

```diff
@@ -0,0 +1,44 @@
+.SplashScreen-root {
+  background-color: $color-text-grey;
+  color: $color-text-light;
+  width: 100%;
+  height: 80vh;
+}
+
+.SplashScreen-content {
```

Comment:
I'd suggest adding `padding: 0 10px;` here to avoid the text/screen-edge problem